code/examples/channel/src/state.rs

```diff
@@ -223,7 +226,11 @@ impl State {
         assert_eq!(round, self.current_round);
 
         // We create a new value.
-        let value = self.make_value();
+        let block = self.fetch_block(height);
```

Comment:
since the block doesn't exist:
```suggestion
        let block = self.make_block(height);
```
also the block should be saved in some UNDECIDED_BLOCK table. And we should stream here the block and not the proposal (as it's done now [here](https://github.com/informalsystems/malachite/blob/fd54d2a9f028fbec645b8b432c8c840f991e9f14/code/examples/channel/src/app.rs#L90)). The `ProposalPart` in context is also wrong now. And reusing the test protos (`ProposalData` should be something like `Transactions transactions ..` or bytes) doesn't work.